image-builder/image_builder.sh

```diff
@@ -89,31 +91,31 @@ IMAGE_DIR=$(readlink -f ${IMAGE_DIR})
 IMAGE_NAME=$(basename ${IMAGE})
 
 if [ -n "${USE_DOCKER}" ] ; then
-	image_name="image-builder-osbuilder"
-
-	docker build  \
-		--build-arg http_proxy="${http_proxy}" \
-		--build-arg https_proxy="${https_proxy}" \
-		-t "${image_name}" "${script_dir}"
-
-	#Make sure we use a compatible runtime to build rootfs
-	# In case Clear Containers Runtime is installed we dont want to hit issue:
-	#https://github.com/clearcontainers/runtime/issues/828
-	docker run  \
-		--rm \
-		--runtime runc  \
-		--privileged \
-		--env IMG_SIZE="${IMG_SIZE}" \
-		--env AGENT_INIT=${AGENT_INIT} \
-		-v /dev:/dev \
-		-v "${script_dir}":"/osbuilder" \
-		-v "${script_dir}/../scripts":"/scripts" \
-		-v "${ROOTFS}":"/rootfs" \
-		-v "${IMAGE_DIR}":"/image" \
-		${image_name} \
-		bash "/osbuilder/${script_name}" -o "/image/${IMAGE_NAME}" /rootfs
-
-	exit $?
+    image_name="image-builder-osbuilder"
+
+    docker build  \
+        --build-arg http_proxy="${http_proxy}" \
```

Comment:
woot  spaces ?

Reply:
Heh, there was a mix of spaces and tabs in the script. I updated it to use spaces,  most of it was spaces indented (`usage()` stayed with tabs cause it's not code indentation). Let's see what other says about it.

Reply:
I normally use tabs, everywhere. I don't think we actually have a project guideline, apart from probably an unwritten 'follow what exists' style.

Reply:
/me grabs :popcorn: to watch the :fireworks:... :smile: